wdl_runner/cromwell_launcher/setup.sh

```diff
@@ -66,7 +101,7 @@ chmod u+x /wdl_runner/wdl_runner.sh
 # Copy Cromwell and the Cromwell conf template
 mkdir /cromwell
 (cd /cromwell && \
-    curl -L -O https://github.com/broadinstitute/cromwell/releases/download/0.19/cromwell-0.19.jar)
+    retry_cmd 'curl -L -O https://github.com/broadinstitute/cromwell/releases/download/0.19/cromwell-0.19.jar')
```

Comment:
It's already at [version 19.3](https://github.com/broadinstitute/cromwell/releases/download/0.19.3/cromwell-0.19.3.jar), it might be better to work of the latest release:

```Bash
$  curl https://github.com/broadinstitute/cromwell/releases | grep jar | grep strong | cut -f2 -d\< | cut -f2 -d\> | head -n1
cromwell-0.19.3.jar
$
```